File: zero/Maruko.Zero/Internal/Service/Role/DTO/RoleMenuDTO.cs
using System.Collections.Generic;

namespace Maruko.Zero
{
    public class RoleMenuDTO
    {
        public long Id { get; set; }
        public long ParentId { get; set; }
        public string Title { get; set; }
        public string Icon { get; set; }

        public string Path { get; set; }
        public string Key { get; set; }

        public List<RoleMenuDTO> Children { get; set; } = new List<RoleMenuDTO>();

        public string Operates { get; set; }
    }
}

File: act/act-modules/act-modules-zero/src/main/java/com/act/modules/zero/internal/application/role/dto/RoleMenuDTO.java
package com.act.modules.zero.internal.application.role.dto;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class RoleMenuDTO {
    private long id;
    private long parentId;
    private String title;
    private String icon;
    private String path;
    private String key;
    private List<RoleMenuDTO> children= new ArrayList<>();
    private String operates;
}
